doc/user/content/integrations/_index.md

```diff
@@ -216,7 +216,7 @@ The level of support for these tools will improve as we extend the coverage of `
 | Preset             | {{< supportLevel alpha >}}       | Connect using the [PostgreSQL database driver](https://superset.apache.org/docs/databases/postgres/).                                                                                                                                                                                               | [](#notify) |
 | Looker             | {{< supportLevel alpha >}}       | Connect using the [PostgreSQL database driver](https://superset.apache.org/docs/databases/postgres/).                                                                                                                                                                                               | [](#notify) |
 | Google Data Studio | {{< supportLevel alpha >}}       | Connect using the [PostgreSQL database driver](https://support.google.com/datastudio/answer/7288010?hl#how-to-connect-to-postgresql&zippy=%2Cin-this-article).                                                                                                                                      | [](#notify) |
-| Tableau            | {{< supportLevel alpha >}}       | Connect using the [PostgreSQL database driver](https://help.tableau.com/current/pro/desktop/en-us/examples_postgresql.htm). Use the JDBC-based driver, which is included by default on Windows but requires an extra [driver download](https://www.tableau.com/support/drivers) on macOS and Linux. | [](#notify) |
+| Tableau            | {{< supportLevel alpha >}}       | Connect using the [PostgreSQL database driver](https://help.tableau.com/current/pro/desktop/en-us/examples_postgresql.htm). See the [Tableau integration page](/integrations/tableau/) for more details. | [](#notify) |
```

Comment:
One of the versions I pushed left a note about the JDBC driver. If possible, I think it'd be great to call out that there is complexity here, as the default PostgreSQL driver definitively does _not_ work with Materialize. Might even be better not to mention "use the PostgreSQL driver" at all here, and instead just say "See the Tableau integration page."

```suggestion
| Tableau            | {{< supportLevel alpha >}}       | Connect using the [PostgreSQL database driver](https://help.tableau.com/current/pro/desktop/en-us/examples_postgresql.htm). Use the JDBC-based driver. See the [Tableau guide](/integrations/tableau) for more details. | [](#notify) |
```